### ISSUE TITLE
Nodes receiving their own 'OUT' in multinode.py

### ISSUE THREAD

**Issue by zlgunn** [pirate/mesh-networking#6]
It seems to me that when a node spits something onto a link ('OUT') it shouldn't receive its own message, though from what I'm seeing here it does.
```
<l0> ready.
Introducing 3 antisocial nodes to the party.
Let there be life.
[n0]:[<l0>]
[n1]:[<l0>]
[n2]:[<l0>]
Type a nodename or linkname to send messages.
        e.g. [$]:n35
             [n35]<en1> ∂5:hi
        or
             [$]:l5
             <l5>(3) [n1,n4,n3]:whats up
    WARNING: ROUTING IS NOT IMPLEMENTED RIGHT NOW, EVERY NODE IS CONNECTED TO EVERY LINK (THIS IS A BUG)
[$]:l0
<l0>(3) [[n0], [n1], [n2]]:hey
[n2] IN  hey[n0] IN  hey

[n1] IN  hey
[$]:l0
<l0>(3) [[n0], [n1], [n2]]:hey
[n0] IN  hey
 [n1] IN  hey
[n2] IN  hey
[$]:n0
[n0]<[<l0>]> ∂2:hi
[n0] OUT ('hi', ('255.255.255.255', 'l0'))
[n0] IN  hi
[n2] IN  hi
 [n1] IN  hi
```

**Comment by pirate** [comment]
Yup, because the link just rebroadcasts it back to everyone who's connected.  Never got around to fixing that bug, maybe I will sometime.

**Comment by pirate** [comment]
Fixed in 6f56b6b
